tests/integration/subchunkable/test_subchunkable.py

```diff
@@ -236,3 +237,48 @@ def test_subchunkable_chunk_ids(clear_temp_dir_and_info_cache):
 
     COLLECTED_CHUNK_IDS.sort()
     assert COLLECTED_CHUNK_IDS == list(range(0, 64))
+
+
+@pytest.mark.skipif(
+    "not config.getoption('--run-integration')",
+    reason="Only run when `--run-integration` is given",
+)
+def test_subchunkable_padded_chunk_ids(clear_temp_dir_and_info_cache):
+    global COLLECTED_CHUNK_IDS  # pylint: disable=global-statement
+    COLLECTED_CHUNK_IDS = []
+    dst_resolution = [16, 16, 42]
+    dst = build_cv_layer(
+        path="file://~/zetta_utils_temp/",
+        info_reference_path="gs://dkronauer-ant-001-synapse/test/inference20240329023545",
```

Comment:
This should point to something in assets, and edited to not need `info_fields_overrides` or `add_scales`.

Reply:
Also, the path should not be in home - you should make a scratch folder in assets.